app/src/main/java/de/westnordost/streetcomplete/screens/main/MainFragment.kt

```diff
@@ -265,6 +270,7 @@ class MainFragment :
     override fun onMapInitialized() {
         binding.gpsTrackingButton.isActivated = mapFragment?.isFollowingPosition ?: false
         binding.gpsTrackingButton.isNavigation = mapFragment?.isNavigationMode ?: false
+        binding.stopTracksButton.isVisible = mapFragment?.tracksRecording ?: false
```

Comment:
This ensures that if the user exits and the app view is re-created, the recording button will show if the recording is still active.